### ISSUE TITLE
[a11y]: ai-chat should show confirmation message after sending feedback

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#908]
### Browser

Chrome

### Operating System

MacOS

### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Description

After filling in the dialog and clicking the Submit button, there is no status message or popup displayed to let users know their feedback was received.
Same issue appears for all feedback instances.

Recommended fix: Ensure a status message or popup appears after submitting feedback to inform users that their response has been successfully received.

User impact: Users, especially those using screen readers or keyboard navigation, may be unsure if their feedback was submitted. This can cause confusion.

Level of Compliance: AA

Severity: Medium

Test tool: Keyboard_Firefox

### WCAG 2.1 Violation

WCAG Mapping: 4.1.3 Status Messages

### Reproduction/example

n/a

### Steps to reproduce

1. open ai-chat
2. ask question
3. wait for answer
4. give feedback on answer
5. observe that no confirmation dialog is shown
Note that the tooltip "feedback sent" is not sufficient to let for example JAWS users know that the feedback was sent.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
